packages/core/test/unit/html/linkProcessor.test.js

```diff
@@ -35,6 +35,83 @@ const mockConfig = {
   ],
 };
 
+test('Test converting .md to .html', () => {
```

Comment:
Since the logic is the same, could use jest's `test.each` here

Reply:
I've updated the test cases to use `test.each`. However, it comes at a cost of slightly less readable test cases. I tried to alleviate it by adding a descriptive test comment above each test case array. What do you think about it? @ang-zeyu